lib/kubeclient/common.rb

```diff
@@ -172,7 +172,9 @@ def get_entities(entity_type, klass, options)
             result.fetch('metadata', {}).fetch('resourceVersion', nil)
       end
 
-      collection = result['items'].map { |item| new_entity(item, klass) }
+      # result['items'] might be nil due to https://github.com/kubernetes/kubernetes/issues/13096
+      collection = []
+      collection = result['items'].map { |item| new_entity(item, klass) } if result['items']
```

Comment:
@moolitayer I think that just:

    collection = result['items'].to_a.map { |item| new_entity(item, klass) } if result['items']

would do the trick.